### PR TITLE
Qur786/#68 mirgate vapor thermal conductivity bacth4(final)

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Comment by qur786** [comment]
@maneetgoyal sir please , don't merge this batch after reviewing, batch 3 still needs to be reviewed by @saikat-samanta

File: test/calculator.test.ts
Comment (maneetgoyal) [inline test/calculator.test.ts:293]: How much difference/variation are we getting in these tests?

Comment (qur786) [reply test/calculator.test.ts:293]: for like 0.5+ to 1+ @maneetgoyal

**Comment by saikat-samanta** [comment]
@qur786, tests look okay. But resolve the conflict.

File: test/calculator.test.ts
Comment (saikat-samanta) [reply test/calculator.test.ts:293]: @qur786, why these tests are commented out?

Comment (qur786) [reply test/calculator.test.ts:293]: those tests are failing I have mentioned these issue with @maneetgoyal sir

**Review by saikat-samanta — APPROVED** [review]

**Comment by maneetgoyal** [comment]
@qur786 The issue was with FormicAcid claculation function. Should be working now.

**Review by maneetgoyal — APPROVED** [review]

**Comment by qur786** [comment]
> @qur786 The issue was with FormicAcid claculation function. Should be working now.

Ok @maneetgoyal sir , I will have a look at it then merge it

**Pull request merged**
